src/core/server/config/deprecation/core_deprecations.test.ts

```diff
@@ -93,6 +93,25 @@ describe('core deprecations', () => {
     });
   });
 
+  describe('server.cors', () => {
+    it('renames server.cors to server.cors.enabled', () => {
+      const { migrated } = applyCoreDeprecations({
+        server: { cors: true },
+      });
+      expect(migrated.server.cors).toEqual({ enabled: true });
+    });
```

Comment:
As `rewriteCorsSettings` is using 'custom' code, can we add a test to assert that passing 

```
{
        server: { cors: { enabled: true },
}
```

does not 'trigger' the deprecation / log message?